google/resource_dataproc_cluster.go

```diff
@@ -717,6 +717,7 @@ func resourceDataprocClusterUpdate(d *schema.ResourceData, meta interface{}) err
 	region := d.Get("region").(string)
 	clusterName := d.Get("name").(string)
 	timeoutInMinutes := int(d.Timeout(schema.TimeoutUpdate).Minutes())
+	gracefulDecommissionTimeout := d.Get("gracefulDecommisionTimeout").(string)
```

Comment:
In order to read this property, it needs to be in the schema defined at the top of the file